db/indexes.go

```diff
@@ -63,32 +70,44 @@ const (
 var (
 	// Simple index names - input to indexNameFormat
 	indexNames = map[SGIndexType]string{
-		IndexAccess:     "access",
-		IndexRoleAccess: "roleAccess",
-		IndexChannels:   "channels",
-		IndexAllDocs:    "allDocs",
-		IndexTombstones: "tombstones",
-		IndexSyncDocs:   "syncDocs",
+		IndexAccess:                 "access",
```

Comment:
We want to create either IndexSyncDocs or the new set of indexes for a given environment - need to discuss the best approach to do this.